grpc-ingest/src/ingester.rs

```diff
@@ -161,258 +141,46 @@ pub async fn run_v2(config: ConfigIngester) -> anyhow::Result<()> {
             })
         })
         .start()?;
+    let snapshot_stream = IngestStream::build()
+        .config(config.accounts.clone())
```

Comment:
```suggestion
        .config(config.snapshots.clone())
```

Needs to read from dedicated config which needs to also be added to the ingester config.